### PR TITLE
K8SPG-597 - Update PG operator chart for 2.4.0 release

### PR DESCRIPTION
**Pull request by jvpasinatto**

### CODE REVIEW

File: charts/pg-db/README.md [hunk 1]
```diff
@@ -24,17 +24,17 @@ To install the chart with the `pg` release name using a dedicated namespace (rec
...
 | `finalizers`                     | Finalizers list                                                 | `{}`                                                        |
-| `crVersion`                     | CR Cluster Manifest version                                                   | `2.3.1`                                                     |
+| `crVersion`                     | CR Cluster Manifest version                                                   | `2.4.0`                                                     |
 | `repository`                     | PostgreSQL container image repository                                               | `percona/percona-postgresql-operator`                       |
-| `image`                     | Postgres image                                                  | `percona/percona-postgresql-operator:2.3.1-ppg16-postgres`  |
+| `image`                     | Postgres image                                                  | `percona/percona-postgresql-operator:2.4.0-ppg16-postgres`  |
```
Comment (tplavcic) [inline charts/pg-db/README.md:37]: please use the image from the cr.yaml

File: charts/pg-db/README.md [hunk 2]
```diff
@@ -122,7 +129,7 @@ The chart can be customized using the following configurable parameters:
 | `backups.restore.enabled`                         |  Enables or disables restoring a previously made backup                                               | `false`                                                     |
 | `backups.restore.repoName`                         | Name of the pgBackRest repository that contains the backup to be restored                                                | `repo1`                                                     |
 | `backups.restore.options`                         | The pgBackRest command-line options for the pgBackRest restore command                                                | `--type=time`                                               |
-| `backups.pgbackrest.image`                     | Set this variable if you need to use a custom pgBackrest image                                        | `percona/percona-postresql-operator:2.3.1-ppg16-pgbackrest` |
+| `backups.pgbackrest.image`                     | Set this variable if you need to use a custom pgBackrest image                                        | `percona/percona-postresql-operator:2.4.0-ppg16-pgbackrest` |
```
Comment (tplavcic) [inline charts/pg-db/README.md:132]: please use the image from the cr.yaml

File: charts/pg-db/README.md [hunk 3]
```diff
@@ -143,6 +150,7 @@ The chart can be customized using the following configurable parameters:
 | `proxy.pgBouncer.tolerations.operator`                    | The Kubernetes Pod tolerations operator for the PostgreSQL instance                                      | `Equal`                                                     |
 | `proxy.pgBouncer.tolerations.value`                       |  The Kubernetes Pod tolerations value for the PostgreSQL instance
 | `proxy.pgBouncer.customTLSSecret.name`                         | Custom external TLS secret name                                                | `keycloakdb-pgbouncer.tls`                                  |
+| `proxy.pgBouncer.securityContext`                           | The Kubernetes Pod security context for the pgBouncer instance | `{}` |
 | `proxy.pgBouncer.affinity.podAntiAffinity` |  Pod anti-affinity, allows setting the standard Kubernetes affinity constraints of any complexity                    | `{}`                                                        |
 | `proxy.pgBouncer.image`                     | Set this variable if you need to use a custom pgbouncer image                                        | `percona/percona-postresql-operator:1.4.0-ppg14-pgbouncer`  |
```
Comment (tplavcic) [inline charts/pg-db/README.md:155]: please fix the pgbouncer image

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: charts/pg-db/values.yaml [hunk 4]
```diff
@@ -289,6 +461,12 @@ pmm:
 #      cpu: 500m
 
 # patroni:
+#   # Some values of the Liveness/Readiness probes of the patroni container are calulated using syncPeriodSeconds by the following formulas:
+#   # - timeoutSeconds:   syncPeriodSeconds / 2;
+#   # - periodSeconds:    syncPeriodSeconds;
+#   # - failureThreshold: leaderLeaseDurationSeconds / syncPeriodSeconds.
+#   syncPeriodSeconds: 10 # default: 10
```
Comment (tplavcic) [inline charts/pg-db/values.yaml:468]: Let's remove the `# default: 10` part

File: charts/pg-db/values.yaml [hunk 5]
```diff
@@ -289,6 +461,12 @@ pmm:
...
+#   leaderLeaseDurationSeconds: 30 # default: 30
```
Comment (tplavcic) [inline charts/pg-db/values.yaml:469]: Let's remove the `# default: 30` part

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Review by ptankov — APPROVED** [review]

**Review by tplavcic — APPROVED** [review]

**Pull request merged**